### PR TITLE
Add option for showdown configuration

### PR DESCRIPTION
**Pull request by Hypercubed**

### CODE REVIEW

File: md.js [hunk 1]
```diff
@@ -2,6 +2,15 @@ var showdown = require('showdown');
 var converter = new showdown.Converter();
 
 exports.translate = function(load) {
+  var converterOptions = this.mdOptions;
```
Comment (guybedford) [inline md.js]: Instead of `this.mdOptions` I think we can use `load.metadata.mdOptions` here to use the metadata pipeline.

**Comment by guybedford** [comment]
Thanks, sure I'd be open to including this.

**Comment by Hypercubed** [comment]
I moved `mdOptions` to metadata.   Much cleaner, I didn't know that was possible.

**Comment by guybedford** [comment]
Thanks, I've published the release.

**Comment by Hypercubed** [comment]
Does this only work when `System.defaultJSExtensions = false;`?

**Comment by Hypercubed** [comment]
We should probably add a note and example to the readme.  Before doing that what is the recommended way to add metadata.  For the test I did:

```
    meta: {
      '*.md': {
        loader: './md.js',
        mdOptions: {
          'tables': true
        }
      }
    }
```

but in my project I needed:

```
  packages: {
    '': {
      defaultJSExtensions: true,
      meta: {
        "*.md": {
          loader: "md",
          mdOptions: {
            tables: true
          }
        }
      }
    }
  },
```

**Comment by guybedford** [comment]
Packages with an actual value is definitely the recommended pattern. See for example https://github.com/frankwallis/plugin-typescript.

**Comment by guybedford** [comment]
(A doc PR would be great, thanks for thinking of this)

**Pull request merged**